presto-hive/src/main/java/com/facebook/presto/hive/LocationService.java

```diff
@@ -38,8 +38,9 @@
     WriteInfo getTableWriteInfo(LocationHandle locationHandle);
 
     /**
-     * Returns {@code WriteInfo} for existing partition partition if {@code partition} is present,
-     * otherwise, returns {@code WriteInfo} for the new partition.
+     * <li> if {@code partition} is present, returns {@code WriteInfo} for appending existing partition
```

Comment:
A few problems here:

* `<li>` must be inside `<ul>` or `<ol>`
* The first sentence of javadoc must be a summary (and therefore cannot be a list).

See `Optional.get` for inspiration.